src/filesystem.cpp

```diff
@@ -309,44 +310,47 @@ AsyncIOContext *FileAccess::asyncfopen(string *f)
 
 bool FileAccess::asyncopenf()
 {
+    numAsyncReads++;
     if (!localname.size())
     {
         return true;
```

Comment:
I guess the answer is *no*, but... would it be possible to reach this point with `isAsyncOpened = false`??